frontend/src/locales/de.json

```diff
@@ -185,7 +185,6 @@
         "title": "Person einladen"
       },
       "collaboratorEdit": {
-        "cannotAssignAnotherRoleToLastManager": "Du kannst der letzten Person mit administrativen Rechten keine andere Rolle zuteilen.",
```

Comment:
this was just another leftover translation?

Reply:
Yes, its not used